docs/faq/troubleshooting.txt

```diff
@@ -34,6 +34,40 @@ To do this, open Terminal.app and navigate (using the ``cd`` command) to the
 directory where :doc:`django-admin </ref/django-admin>` is installed, then
 run the command ``sudo chmod +x django-admin``.
 
+.. _troubleshooting-unit-testing:
+
+Unit Testing Problems
+=============================
+
+Installing test suite dependencies fails
+----------------------------------------
+
+Some of test cases rely on external libraries, which are required by these tests
+in order to work properly. These libraries are dependencies.
+
+When installing :ref:`these <running-unit-tests-dependencies>` dependencies with 
+``pip install tests/requirements/py*.txt`` command you might run across
+the following error:
+
+.. code-block:: console
+
+      ...
+    $ Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-PCA0QU/argon2-cffi/
+
+On the other hand dependency is built using a few other libraries. 
+Lacking any of the latter may cause you problems like the one above.
+
+The best way to deal with such errors, when installing test suite dependencies,
+is to check the dependency's documentation in search for required libraries.
+
+In our case we were missing *libffi* library used in 
+`argon2-cffi <https://argon2-cffi.readthedocs.io/en/stable/installation.html>`_.
+All you need to do is basically install the missing library using pip.
```

Comment:
I'd leave out "basically" here. Either it's all you need to do or it's not. Also, is `libffi-dev` pip-installable? My guess is it is not.